content/en/docs/setup/install/operator/index.md

```diff
@@ -93,6 +93,26 @@ EOF
 The controller will detect the `IstioOperator` resource and then install the Istio
 components corresponding to the specified (`demo`) configuration.
 
+If you have used `--watchedNamespaces` to initialize the Istio operator, make sure the IstioOperator resource is located in the specified namespace list.
+
+When installing Istio in the namespace other than `istio-system`, specify the namespace with `values.global.istioNamespace` as the following.
+
+{{< text bash >}}
+$ kubectl create ns istio-namespace1
+$ kubectl apply -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-namespace1
+  name: example-istiocontrolplane
+spec:
+  profile: demo
+  values:
+    global:
+      istioNamespace: istio-namespace1
+EOF
+{{< /text >}}
```

Comment:
This example is confusing, it's conflating watchedNamespaces with istioNamespace. I suggest just showing the istioNamespace part in the example:
```suggestion
{{< text yaml >}}
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
...
spec:
  profile: demo
  values:
    global:
      istioNamespace: istio-namespace1
{{< /text >}}
```